crux-llvm/src/Crux/LLVM/Config.hs

```diff
@@ -107,6 +108,9 @@ llvmCruxConfig =
          optLevel <- Crux.section "opt-level" Crux.numSpec 1
                            "Optimization level to request from `clang`"
 
+         loopMerge <- Crux.section "opt-loop-merge" Crux.yesOrNoSpec False
+                        "Insert merge blocks in loops with early exits"
```

Comment:
Maybe add a short sentence here about why you'd want to do this.  "may speed up simulation", etc.